IBM_CLOUD_DEPLOY.md

```diff
@@ -0,0 +1,29 @@
+# Deploying to IBM Cloud
+
+1. Create a cluster using the [IBM Cloud Container Service](https://console.bluemix.net/containers-kubernetes/clusters
+). For this Code Pattern we will only need to use the Free tier which gives us access to a Kubernetes cluster with 1 worker node. (Could take anywhere from 10 minutes to half an hour.)
+
+![Create Cluster](doc/source/images/create-cluster.png)
+
+2. (Optional Step) The instructions to install the command line tools are optional and not required to deploy the model using this tutorial. You may choose to install them while waiting for the cluster to be created, if you wish to do so.
+
+3. On the IBM Dashboard select `Worker Nodes` and copy the `Public IP Address` for the node.
```

Comment:
Maybe specify how to get to the IBM Dashboard

Reply:
Also since there are many step after this I'd say "Take not of..." rather than "copy" the IP

Reply:
👍